packages/odyssey-react/src/components/Radio/RadioButton.theme.ts

```diff
@@ -13,33 +13,36 @@
 import type { ThemeReducer } from "@okta/odyssey-react-theme";
 
 export const theme: ThemeReducer = (theme) => ({
-  ColorBorder: theme.ColorBorderUi,
-  ColorBorderHover: theme.ColorPrimaryBase,
-  ColorBorderFocus: theme.ColorPrimaryBase,
-  ColorBorderChecked: theme.ColorPrimaryBase,
-  ColorBorderDisabled: theme.ColorBorderDisabled,
-  ColorBorderDisabledChecked: theme.ColorPrimaryLight,
-  ColorBorderInvalid: theme.ColorBorderDangerBase,
-  ColorBorderInvalidHover: theme.ColorDangerDark,
-  ColorBackground: "transparent",
-  ColorBackgroundChecked: theme.ColorPrimaryBase,
-  ColorBackgroundDisabled: theme.ColorBackgroundDisabled,
-  ColorBackgroundDisabledChecked: theme.ColorPrimaryLight,
-  ColorBackgroundInvalid: theme.ColorBackgroundDangerDark,
-  ColorDot: theme.ColorTextBodyInverse,
-  ColorFocusOutline: theme.ColorFocusPrimary,
-  ColorFocusOutlineInvalid: theme.ColorFocusDanger,
-  FocusOutlineWidth: theme.FocusOutlineWidthSmall,
-  ColorTextDisabled: theme.ColorTextBody,
-  ColorTextInvalid: theme.ColorTextDanger,
-  MarginBlockEnd: theme.SpaceRemXs,
-  LabelPaddingInlineStart: theme.SpaceEmS,
-  FontSize: theme.FontSizeBody,
-  FontWeight: theme.FontWeightNormal,
+  CircleBackgroundColor: "transparent",
+  CircleBorderColor: theme.ColorBorderUi,
+  CircleCheckedBackgroundColor: theme.ColorPrimaryBase,
+  CircleCheckedBorderColor: theme.ColorPrimaryBase,
+  CircleDisabledBackgroundColor: theme.ColorBackgroundDisabled,
+  CircleDisabledBorderColor: theme.ColorBorderDisabled,
+  CircleDisabledCheckedBackgroundColor: theme.ColorPrimaryLight,
+  CircleDisabledCheckedBorderColor: theme.ColorPrimaryLight,
+  CircleDotBorderStyle: theme.BorderStyleBase,
+  CircleDotBorderWidth: "2px",
```

Comment:
Small detail, but it's just the Circle (not the Dot) that has border width and style

Reply:
Thanks will hit this in a follow up PR making a note

Reply:
[@andrewberg-okta the styles do appear to be shared across both pseudo-elements however.](https://github.com/okta/odyssey/blob/develop/packages/odyssey-react/src/components/Radio/RadioButton.module.scss#L115-L130)

Reply:
cc @andrewberg-okta please let me know about the above ☝️ I believe the token name as shown in the PR is correct (should be `CircleDot` )